Add `Indonesia Convert Cup Open 2022` wiki article (#8714)
ppy/osu-wiki: wiki/Tournaments/ICC/2022/en.md

---
tags:
  - ICCO2022
  - ICCO 2022
---

# Indonesia Convert Cup Open 2022

![Indonesia Convert Cup Open 2022 logo](img/logo.jpg)

> The **Indonesia Convert Cup 2022** (***ICC 2022***) was a double-elimination 1v1 osu!catch tournament hosted by ::{ flag=ID }:: [Urabe Mikoto](https://osu.ppy.sh/users/1272422) in which all the beatmaps were osu! converts. Despite its name, the tournament was open to all players from all ten [ASEAN member states](https://asean.org/about-asean/member-states/) as well as ::{ flag=TL }:: Timor-Leste, ::{ flag=NZ }:: New Zealand, and ::{ flag=AU }:: Australia with 3,000 playcounts or more in the osu!catch game mode to their name. It was regarded as the third instalment of the Indonesia Convert Cup tournament series.

Reviewer: ```suggestion
The **Indonesia Convert Cup Open 2022** (***ICCO 2022***) was a double-elimination 1v1 osu!catch tournament hosted by ::{ flag=ID }:: [Urabe Mikoto](https://osu.ppy.sh/users/1272422) in which all the beatmaps were osu! converts. Despite its name, the tournament was open to all players from all ten [ASEAN member states](https://asean.org/about-asean/member-states/) as well as ::{ flag=TL }:: Timor-Leste, ::{ flag=NZ }:: New Zealand, and ::{ flag=AU }:: Australia with 3,000 play count or more in the osu!catch game mode to their name. It was regarded as the third instalment of the Indonesia Convert Cup tournament series.
```

| Graphic designer | ::{ flag=RU }:: [ThankYou](https://osu.ppy.sh/users/4571241) |
| Spreadsheet manager | ::{ flag=ID }:: [Madoka Ayukawa](https://osu.ppy.sh/users/1595221) |
| Sponsor | ::{ flag=ID }:: [-Hex-](https://osu.ppy.sh/users/8630988), ::{ flag=MY }:: [Moeruattack](https://osu.ppy.sh/users/4480237) |
| Wiki editor | ::{ flag=ID }:: [Niva](https://osu.ppy.sh/users/197805) |

## Links

- **[Main spreadsheet](https://docs.google.com/spreadsheets/u/1/d/e/2PACX-1vRLbMRr5JVXAYJx5l7DY-Vecwo7ESaLgmFieo8rQwbAqlmC_o9SRo072JadnxcSzAbG0eWncroKBq91/pubhtml#)**
- [Forum thread](https://osu.ppy.sh/community/forums/topics/1649090)
- [Discord server](https://discord.gg/7g6TAj2)
- [Challonge brackets](https://challonge.com/ICCO_2022)
- [Livestream channel](https://www.twitch.tv/osucatchid)

## Participants

Listed below are the players who managed to qualify into the Round of 32 (along with their seed number and average map rank combined across each Qualifier beatmap) out of 39 registered players in total.

> | Seed No. | Player | Avg. Map Rank |

Reviewer: ```suggestion
| Seed no. | Player | Avg. map rank |
```

| 18 | ::{ flag=TH }:: [Kukkai](https://osu.ppy.sh/users/7811952) | 17.88 |
| 19 | ::{ flag=ID }:: [Mamat](https://osu.ppy.sh/users/3031177) | 17.88 |
| 20 | ::{ flag=AU }:: [Geniance](https://osu.ppy.sh/users/4229240) | 18.63 |
| 21 | ::{ flag=TH }:: [RAISE MY SWORD](https://osu.ppy.sh/users/17385454) | 19.75 |
| 22 | ::{ flag=ID}:: [Mixuri](https://osu.ppy.sh/users/9153772) | 19.88 |
| 23 | ::{ flag=MY }:: [Asteristia](https://osu.ppy.sh/users/9434933) | 21.00 |
| 24 | ::{ flag=MY }:: [DTNLRiki](https://osu.ppy.sh/users/12382410) | 22.75 |
| 25 | ::{ flag=ID }:: [Zvenx](https://osu.ppy.sh/users/14613788) | 23.13 |
| 26 | ::{ flag=ID }:: [Mochi -](https://osu.ppy.sh/users/20424806) | 23.13 |
| 27 | ::{ flag=ID }:: [Kinora](https://osu.ppy.sh/users/20571283) | 26.25 |
| 28 | ::{ flag=PH }:: [-HyperDream](https://osu.ppy.sh/users/12410693) | 26.25 |
| 29 | ::{ flag=ID }:: [Fisicia](https://osu.ppy.sh/users/6556054) | 26.63 |
| 30 | ::{ flag=PH }:: [-Isla-](https://osu.ppy.sh/users/17745759) | 29.13 |
| 31 | ::{ flag=PH }:: [Chloebe](https://osu.ppy.sh/users/4577865) | 29.25 |
| 32 | ::{ flag=ID }:: [Cera-](https://osu.ppy.sh/users/1974131) | 29.75 |

## Podium

> ![](img/podium.jpg "ICCO 2022 podium")

Reviewer: ```suggestion
![ICCO 2022 podium](img/podium.jpg "ICCO 2022 podium")
```

     - Be in possession of a valid identity document (i.e. passport or ID card) issued by one of the aforementioned countries.
2. To ensure that all incoming registrations are serious and valid, every registered player will be checked in detail by the tournament management.
3. The list of players who are deemed to be eligible to compete in the tournament will be published by the tournament management after the registration phase has ended.
4. Testplayers, referees, and mappool selectors may not participate as players in this tournament.

### Round-specific rules

#### Qualifier rules

1. Each player will have to sign up to one of the Qualifier lobbies that have been scheduled and prepared by the tournament management in advance.

> 2. In the lobby, all players will have to consecutively play all of the nine Qualifier beatmaps in the order of NM1 -> NM2 -> HD1 -> HD2 -> HR1 -> HR2 -> DT1 -> DT2.

Reviewer: i count 8 in that list

```suggestion
2. In the lobby, all players will have to consecutively play all of the eight Qualifier beatmaps in the order of NM1 -> NM2 -> HD1 -> HD2 -> HR1 -> HR2 -> DT1 -> DT2.
```

3. Players **are not allowed** to ban any beatmaps in the Qualifiers.

> 4. Players **are not allowed** to join in (or register for) more than one Qualifier lobby.

Reviewer: ```suggestion
4. Players **are not allowed** to join (or register for) more than one Qualifier lobby.
```

   - In both of the Finals weeks, each player needs to win 7 points in order to win a match. (Best of 13)
   - Whether there are players who are declared to win the match by default.
   - Whether there are players who are disqualified from the tournament.

### Match instructions

1. Each player must use the `!roll` command once in the multiplayer lobby in order to determine the banning and picking order.
   - The winner of the `!roll` gets to determine who gets the first pick and the second ban.
   - The loser of the `!roll` gets the opposite by default.
   - This rule does not apply in the Qualifier lobbies.
2. Each player has to ban **one beatmap** from the corresponding mappool. These beatmaps will not be allowed to be picked by any player during the entire match.
   - Barring the tiebreaker, there are no restrictions as to which maps may and may not be banned in a match.
   - Banning does not apply in the Qualifier lobbies.
3. Each player is free to select one warm-up beatmap. Using beatmaps with questionable content is prohibited. 
   - Warm-ups do not apply in the Qualifier lobbies.

> 4. **There will be no dedicated FreeMod picks in the tournament.** However, when playing a map from the HardRock or Double Time pool, Hidden can be used optionally on top of the corresponding mod by any player.

Reviewer: ```suggestion
4. **There will be no dedicated FreeMod picks in the tournament.** However, when playing a map from the HardRock or DoubleTime pool, Hidden can be used optionally on top of the corresponding mod by any player.
```
